babel.config.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  presets: [
+    [
+      "@babel/preset-env",
+      {
+        modules: false,
```

Comment:
Why we should set modules to false?

Reply:
Because we ship the app to the browser, there is no need to transform from ES syntax to another one, so I set it to false, there is more detail in [this link](https://babeljs.io/docs/en/babel-preset-env#modules)

Reply:
and also it says  

> If you are using a bundler with Babel, the default modules: "auto" is always preferred.